packages/synapse-react-client/src/synapse-queries/wiki/useWikiPage.ts

```diff
@@ -0,0 +1,20 @@
+import { ObjectType, WikiPage } from '@sage-bionetworks/synapse-types'
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { SynapseClientError } from '../../utils/SynapseClientError'
+import { useSynapseContext } from '../../utils'
+import SynapseClient from '../../synapse-client'
+
+export function useWikiPage(
```

Comment:
Is this different from useWiki.ts `useGetWikiPage`?

Reply:
Nice catch.  I pulled in changes from an old PR, perhaps useGetWikiPage was unavailable at that time.  Replaced with useWiki.useGetWikiPage()!